src/SetupModel.cpp

```diff
@@ -1688,7 +1690,7 @@ void SetupAirports(void)
 const double PROP_OTHER_PARENT_AWAY = 0.0;
 
 
-void AssignHouseholdAges(int n, int pers, int tn)
+void AssignHouseholdAges(int n, int pers, int tn, std::string const& reg_demog_file)
```

Comment:
This feels like the wrong interface - we only test `reg_demog_file` for emptiness maybe something like:
```suggestion
void AssignHouseholdAges(int n, int pers, int tn, bool do_adunit_demog)
```
Instead?

Reply:
Done